CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [FEATURE] Query Frontend: Add peakSample in query stats response. #6188
```

Comment:
This should be an enhancement not a feature.

Reply:
I've fixed it.